pkg/kubelet/dockershim/exec_test.go

```diff
@@ -110,23 +143,27 @@ func TestExecInContainer(t *testing.T) {
 	var resize <-chan remotecommand.TerminalSize
 
 	for _, tc := range testcases {
-		t.Logf("TestCase: %q", tc.description)
+		tc := tc
```

Comment:
Why is this line needed?

Reply:
It has to do with the closure provided by the enumeration (`tc` is closed over inside the `testcases` slice) and how the invocation of test.Parallel() can produce variable reference collisions.

This public gist is a nice explanation: https://gist.github.com/posener/92a55c4cd441fc5e5e85f27bca008721#what-happened

However! We're not actually invoking `.Parallel()` :/

Should we run these tests concurrently? If so, I'll add the `.Parallel()` foo, otherwise, yeah, I'll remove that `tc := tc` (it's probably just a habit from writing parallel golang UT).

Reply:
I went ahead and enabled parallelization after doing a cursory audit of the UT landscape. As you'd expect in k/k, there are a million different patterns in the codebase, but I saw enough of these that I felt that the way we're introducing parallelization here is not without precedent. E.g.:

- https://github.com/kubernetes/kubernetes/blob/master/pkg/security/podsecuritypolicy/util/util_test.go#L462
- https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/apimachinery/pkg/runtime/converter_test.go#L586
- https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/client-go/tools/watch/retrywatcher_test.go#L514

(etc)